docs/io/optional/logging_configuration.ipynb

```diff
@@ -0,0 +1,2226 @@
+{
```

Comment:
<p>Wording of the first sentence can be improved slightly.</p><p> </p><p>The specific argument tells the logger to only capture the log messages that are the level set by the log_state parameter.</p><p> </p><p>Add a comma before and after i.e. so it is "input, i.e., "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>